internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,51 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	dataSlice := strings.Split(datastring, ",")
+	if len(dataSlice) != 2 {
+		return fmt.Errorf("invalid input format: expected 2 values, got %d: %s", len(dataSlice), datastring)
+	}
+	steps, err := strconv.Atoi(dataSlice[0])
+	if err != nil {
+		return fmt.Errorf("invalid step count: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("invalid step count: %w", err)
```